frontend/storybook/src/components/entry/imagelabel/admin/images/image-label-left.ts

```diff
@@ -0,0 +1,44 @@
+import "@elements/entry/admin/images/meta/sections/left";
```

Comment:
The file location here should match the storybook - `entry/admin/images/meta/...`